src/updater.js

```diff
@@ -126,6 +127,9 @@ export default class Updater {
       spawn(this.binPath, ['update'], {stdio: [null, fd, fd]})
       .on('error', e => this.out.warn(e, 'autoupdate:'))
     } catch (e) { this.out.warn(e, 'autoupdate:') }
+    try {
+      await logChopper.chop(path.join(this.config.dirs.cache, 'error.log'))
```

Comment:
@RasPhilCo Could you use `errlog` from [Output](https://github.com/heroku/cli-engine-command/blob/master/src/output/index.js#L192)?